### ISSUE TITLE
Font size, typeface, and layout customization options.

### ISSUE THREAD

**Issue by fishyo** [Ranchero-Software/NetNewsWire#4438]
Thank you for creating such an excellent RSS reader. As an iOS user, I've encountered some RSS feeds with excessively small font sizes, making reading cumbersome.  I would therefore request the addition of font size and page layout customization options in future updates. Thank you for your time and consideration.

**Comment by vardecab** [comment]
I find the font size too small to be an issue on the mobile app as well. I tried creating a theme with a larger font size, but it doesn’t work on iOS.

I feel like if this setting were ported from the macOS app to the iOS app, it would solve the problem?
![0WIz9XlT](https://github.com/user-attachments/assets/5423d862-1813-4b6c-9ee6-e4be78ddde92)

**Comment by fanyangto** [comment]
I have the same concern/request - on iOS (iphone, not macOS), there is no way to change the RSS article fonts. It's too small to read. 
The above suggestion is on MacOS, not iOS.

**Comment by vardecab** [comment]
> The above suggestion is on MacOS, not iOS.

It works on both. 

Here you can see one of the default themes - Sepia:
![Image](https://github.com/user-attachments/assets/744b0880-a79f-4cab-bdba-686f6d70fac2)

And here a theme I did for myself:
![Image](https://github.com/user-attachments/assets/f8210300-790e-4d28-8509-1f84ab374df6)

**Comment by fanyangto** [comment]
> > The above suggestion is on MacOS, not iOS.
> 
> It works on both.
> 
> Here you can see one of the default themes - Sepia: ![Image](https://github.com/user-attachments/assets/744b0880-a79f-4cab-bdba-686f6d70fac2)
> 
> And here a theme I did for myself: ![Image](https://github.com/user-attachments/assets/f8210300-790e-4d28-8509-1f84ab374df6)

Now I see why I was confused.

On iphone, the themes are named as a mixture of color and readability, unlike MacOS setting you showed (S/M/L/XL). 

On iPhone, it's named Default/Appanoose/Hyperlegible/Promenade/Sepia). I thought they all change color only. But it's a mixture of color and font, and there is no independent setting for fonts. However, Hyperlegible is good enough for me. So at least my problem is solved. 

I'd think it's more suitable to have independent color and font settings so they can be changed one at a time. 

Thank you for alerting me to look into it deeper. 

Fan
